op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -1,170 +1,175 @@
+// Package loadtest contains interop load tests.
+//
+// Set NAT_INTEROP_LOADTEST_TARGET to the initial amount of messages that should be passed per L2 slot in each test (default: 100).
+//
+// Each test increases the message throughput until some threshold is reached (e.g., the gas target).
+// The throughput is decreased if the threshold is exceeded or if errors are encountered (e.g., transaction inclusion failures).
 package loadtest
 
 import (
-	"math"
 	"math/rand"
 	"os"
+	"path/filepath"
 	"strconv"
 	"sync"
 	"testing"
 	"time"
 
-	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
-	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
 	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
 	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
 	"github.com/ethereum-optimism/optimism/op-devstack/presets"
 	"github.com/ethereum-optimism/optimism/op-service/eth"
-	"github.com/ethereum-optimism/optimism/op-service/plan"
-	"github.com/ethereum-optimism/optimism/op-service/txintent"
-	"github.com/ethereum-optimism/optimism/op-service/txplan"
-	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
-	"github.com/ethereum/go-ethereum/common"
 	"golang.org/x/net/context"
 )
 
 func TestMain(m *testing.M) {
 	presets.DoMain(m, presets.WithSimpleInterop())
 }
 
-// TestLoad passes messages from one chain to another.
-// It assumes that both chains have the same block time.
-// Set NAT_INTEROP_LOADTEST_TARGET to the initial amount of messages that should be passed per block time.
-// The test will run until the test deadline.
-func TestLoad(gt *testing.T) {
+// TestSteady attempts to slightly exceed the gas target in every block by spamming interop messages,
+// simulating benign but heavy activity.
+// Set NAT_INTEROP_LOADTEST_STEADY_TIMEOUT to configure TestSteady's timeout.
```

Comment:
Let me know if you think the env var is too long. For test-specific configs, maybe `NAT_<test-name>_<var-name>` would be a superior convention, as in `NAT_STEADY_TIMEOUT`.